arithmetization/src/test/java/net/consensys/linea/zktracer/instructionprocessing/callTests/sixtyThreeSixtyFourths/SixtyThreeSixtyFourthsTests.java

```diff
@@ -0,0 +1,390 @@
+      Stream.of(
+              ECREC,
+              SHA256,
+              RIPEMD160,
+              ID,
+              MODEXP,
+              ALTBN128_ADD,
+              ALTBN128_MUL,
+              ALTBN128_PAIRING,
+              BLAKE2B_F_COMPRESSION)
+          .collect(
+              Collectors.toMap(
+                  address -> address,
+                  address ->
+                      new HashMap<>() {
+                        {
+                          put(
+                              false,
+                              BytecodeRunner.of(preCallProgram(address, false, false, 0))
+                                  .runOnlyForGasCost(additionalAccounts));
+                          put(
+                              true,
+                              BytecodeRunner.of(preCallProgram(address, false, true, 0))
+                                  .runOnlyForGasCost(additionalAccounts));
+                        }
+                      }));
+
+  // Note: transferValue = false and cds = 0 as we are interested only in the cost of the
+  // corresponding PUSHes here
+
+  @ParameterizedTest
+  @MethodSource("fixedCostLTStipendTestSource")
+  void fixedCostLTStipendTest(long gasLimit, boolean insufficientGasForPrecompileExpected) {
+    // Only ECADD falls in this scenario and whenever transferValue = true, gas is enough
+    // Thus, we only test the case in which transferValue = false
+
+    final BytecodeCompiler program = BytecodeCompiler.newProgram();
+
+    program.immediate(preCallProgram(ALTBN128_ADD, false, false, 0)).op(CALL);
+
+    final BytecodeRunner bytecodeRunner = BytecodeRunner.of(program);
+    bytecodeRunner.run(gasLimit);
+
+    // insufficientGasForPrecompileExpected = true  =>
+    // targetCalleeGas = 63/64 * (252 - 100) = 150
+    // insufficientGasForPrecompileExpected = false =>
+    // targetCalleeGas = 63/64 * (251 - 100) = 149
+
+    assertNotEquals(
+        OUT_OF_GAS_EXCEPTION,
+        bytecodeRunner.getHub().previousTraceSection().commonValues.tracedException());
+
+    final boolean insufficientGasForPrecompileActual =
+        bytecodeRunner.getHub().oob().operations().stream()
+            .anyMatch(OobOperation::isInsufficientGasForPrecompile);
+
+    assertEquals(insufficientGasForPrecompileExpected, insufficientGasForPrecompileActual);
+  }
+
+  Stream<Arguments> fixedCostLTStipendTestSource() {
+    List<Arguments> arguments = new ArrayList<>();
+    final long targetCalleeGas = PrecompileUtils.getECADDCost();
+    for (int cornerCase : List.of(0, -1)) {
+      final long gasLimit =
+          getGasLimit(
+              targetCalleeGas + cornerCase,
+              false,
+              false,
+              preCallProgramGasMap.get(ALTBN128_ADD).get(false));
+      arguments.add(Arguments.of(gasLimit, cornerCase == -1));
+    }
+    return arguments.stream();
+  }
+
+  @ParameterizedTest
+  @MethodSource("costGEQStipendTest")
+  void costGEQStipendTest(
```

Comment:
Here we are testing precompiles whose cost can exceed $G_\text{callstipend}$. Maybe we can rename it something à la 
```java
void prcCallsWithOrWithoutValueTransferWithCalleeGasEitherPreciselyMatchingOrPreciselyFallingShortOfPrcExecutionCost(...) { ... }
```

Reply:
It's a long name but it accurately describes what is being tested 😁

Reply:
Alternatively call it test2 and add a doc string.